src/arrow-sdk.ts

```diff
@@ -188,25 +197,34 @@ export async function getStrikeGrid(
     contractType: number,
     version: VERSION = VERSION.V2
 ) {
-    const strikeGridResponse = await axios.get(
-        urls.api[version] + `/get-strike-grid?ticker=${ticker}&expiration=${readableExpiration}&contract_type=${contractType}`
-    )
-
-    const strikeGrid = []
-    for (let i = 0; i < strikeGridResponse.data.options.length; i++) {
-        const strikeGridOption = strikeGridResponse.data.options[i]
-        const option: Option = {
-            ticker: ticker,
-            expiration: readableExpiration,
-            strike: strikeGridOption.strike,
-            contractType: contractType,
-            price: strikeGridOption.price,
-            greeks:  strikeGridOption.greeks
+    switch(version) {
```

Comment:
This is fine, but maybe the best way to do this is just put a break after `case VERSION.COMPETITION:` and run the code outside of the switch